src/components/shared/playbackButton.js

```diff
@@ -0,0 +1,65 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { graphql } from "gatsby"
+import { contentExists } from "../../utils/ug-utils"
```

Comment:
Remove. We're in the process of decommissioning this function so it not be used in new components.